templates/about.html

```diff
@@ -3,10 +3,9 @@
 {% block body %}
     <div class="jumbotron text-center">
         <h1>About US</h1>
-        <p class="lead">The Bucket is all about giving you the opportunity to maximise your potential and 
-            get to experience all which you have always dreamt of. It's a proprietary service that elps you keep track of all 
-            the exhilirating activities that not many get to experience. Sign up today and give it a spin.
-        </p>
+            <p class="lead">The Bucket is all about giving you the opportunity to maximise your potential and 
+                get to experience all which you have always dreamt of. Sign up today and give it a spin.
+            </p>
```

Comment:
Okay. I'll try and do all that when refactoring. For now I was just trying to get the hang of creating and merging a pull request. :)